subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/UITestConfig.kt

```diff
@@ -45,8 +44,7 @@ object UITestConfig {
      * Exceptions to be waited for; any unregistered exceptions will be propagated
      */
     var waiterAllowedExceptions = setOf(
-        PerformException::class.java,
-        NoMatchingViewException::class.java,
+        EspressoException::class.java,
         AssertionError::class.java
     )
```

Comment:
I've faced with another espresso exception - `AmbiguousViewMatcherException` in e2e tests.
I decided that it's unreliable to duplicate all possible espresso errors here. It's better to use a common marker type.